### ISSUE TITLE
getnewbasecoinaddress doesn't store new address in wallet outside console return 1.0.0.1 (non-test)

### ISSUE THREAD

**Issue by StrontiumZ38** [Veil-Project/veil#159]
### Describe the issue
Console command 'getnewbasecoinaddress' does not store new address anywhere visibly in the wallet outside of the return in console. This should be stored in address book for future reference.

UPDATE
The new address was added to the address book, but only after several minutes and several visits to the page.

#### Can you recreate the problem
Yes
#### If so, please list the steps to reproduce below:

Run wallet. Go into setting, advanced, console, and enter getnewbasecoinaddress

### Expected behavior
Addresses printed from getnewbasecoinaddress in console should be stored somewhere in the wallet for future reference - most ideally the address book.

### Actual behavior
Addresses created in this manner are simply printed in console and not stored elsewhere in the wallet.. 

### What version of Veil wallet are you using?
Binary builds Windows (64) (1.0.0.1)

**Comment by veildev** [comment]
This is not happening on the latest master (at least here) and it's not a GUI issue. If the address is correctly stored on the rpc command then the GUI will show it.

**Comment by StrontiumZ38** [comment]
Still have trouble with this.

**Comment by StrontiumZ38** [comment]
This is fixed, but has created a new bug. Closing this and making new issue
